java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -70,7 +69,7 @@ public ProgramPoint(CFG.Block block, int i) {
 
     @Override
     public int hashCode() {
-      if(hashcode == 0) {
+      if (hashcode == 0) {
```

Comment:
No need for refactoring but those cosmetics changes are cool, but should not be part of such a commit. Makes the diff harder to read for reviewer.